lib/ui.lib.php

```diff
@@ -713,7 +713,7 @@ function toggle_visible($element)
 function display_notification($message, $timeout = 5000)
 {
     echo "<script type='text/javascript'>";
-    echo "displayNotification('" . addslashes(json_encode($message)) . "', " . $timeout . ");";
+    echo "displayNotification('" . addslashes(json_encode($message, JSON_UNESCAPED_UNICODE)) . "', " . $timeout . ");";
```

Comment:
Maybe we should keep the escaped unicode and just replace https://github.com/ampache/ampache/blob/c053d438160817fd80204e4ea15afff07f0e8798/lib/javascript/base.js#L107

by

```
$("#notification-content").html(JSON.stringify(JSON.parse(message)));
```

?